### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by zihadbinjahangir**
I have added some python code to handle the problem of repeatedly adding the same data.

### CODE REVIEW

File: grab_all_data.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+from api_callling.api_calling import APICall
+ticker_info = pd.DataFrame(APICall.client.get_ticker())
+# print(ticker_info)
+fs = FindSymbols()
+all_symbols_payers = fs.get_all_symbols("BUSD", ticker_info)['symbol']
+
+# print(all_symbols_payers)
+# symbol_data_already_collected
+
+try:
```
Comment (sushen) [inline grab_all_data.py:41]: Use if-else instead of try-except.

File: grab_all_data.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+# pd.set_option('display.max_rows', 500)
+# pd.set_option('display.max_columns', 500)
+# pd.set_option('display.width', 1000)
+#
+from api_callling.api_calling import APICall
+ticker_info = pd.DataFrame(APICall.client.get_ticker())
+# print(ticker_info)
+fs = FindSymbols()
+all_symbols_payers = fs.get_all_symbols("BUSD", ticker_info)['symbol']
+
```
Comment (sushen) [inline grab_all_data.py:37]: Use Len to see how many symbols there are each time we re-run the script.

File: grab_all_data.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    symbol_data_already_collected = []
+
+print(symbol_data_already_collected)
+
+# print(input(":"))
+
+for symbol in all_symbols_payers:
+
+    # if press q then the code will be stop here
+    if keyboard.is_pressed('q'):
```
Comment (sushen) [inline grab_all_data.py:56]: This if statement is not needed.

File: grab_all_data.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        break
+
+    if symbol in symbol_data_already_collected:
```
Comment (sushen) [inline grab_all_data.py:59]: Use Sql data to see already downloaded data.

Comment (zihadbinjahangir) [reply grab_all_data.py:59]: We will do this thing on the missing data script.

File: grab_all_data.py (hunk 1 above)
Comment (zihadbinjahangir) [reply grab_all_data.py:42]: We will handle this thing later.

**Review by sushen — APPROVED** [review]

**Pull request merged**